app/design/frontend/Edu/Ultimate/web/assets/scss/component/_header.scss

```diff
@@ -0,0 +1,94 @@
+.page-header{
+  background-color: white;
+}
+.sticky_sections{
+  height: 70px;
+}
+@media (min-width: 769px) {
```

Comment:
scss lets you create variables, why not make variables for different break points?